README.md

```diff
@@ -54,7 +53,7 @@ We can now build the Vehicle Gateway itself, by overlaying its workspace on top
 cd ~/vg/vg_ws
 rosdep update && rosdep install --from-paths src --ignore-src -y
 source ~/vg/gz_ws/install/setup.bash
-colcon build --merge-install --event-handlers console_direct+
+colcon build
```

Comment:
do you have any reason to remove these two options?

for example`--event-handlers console_direct+` will help us if someone has some issues compiling the repo